scope/src/doctor/check.rs

```diff
@@ -467,6 +468,7 @@ impl DefaultDoctorActionRun {
         let mut result: Option<CacheStatus> = None;
 
         for command in &action_command.commands {
+            //FIXME CMAC: we also need to do shell expansion here
```

Comment:
add tests and perform shell expansion here

take a look at if it makes sense to share the command execution code between the check commands and fix commands (I think it might)